addons/armaos/functions/fnc_terminal_setKeyboardLayout.sqf

```diff
@@ -26,4 +26,17 @@ else
 	["AE3_KeyboardLayout", _terminalKeyboardLayout, 0, "client", true] call CBA_settings_fnc_set;
 };
 
-_computer setVariable ["AE3_terminal", _terminal];
\ No newline at end of file
+_computer setVariable ["AE3_terminal", _terminal];
+
+/* ------------- UI on Texture ------------ */
+
+if (AE3_UiOnTexture) then
+{
+	private _playersInRange = [3, _computer] call AE3_main_fnc_getPlayersInRange;
+
+	{
```

Comment:
You can give the broadcast targets directly to remoteExec:
```
[_computer, _terminalKeyboardLayout] remoteExec ["AE3_armaos_fnc_terminal_uiOnTex_setKeyboardLayout", _playersInRange];
```

Reply:
Sounds good. Will do that.